.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please help potential reviewers to understand this pull request and speed up 
+the process by answering the following questions, where appropriate.
+
+# Why was this pull request created?
+
+# How is this achieved?
+
+# How to test this?
```

Comment:
Yeah, but thats why we link bugs to a change (the bug should have reproducer instructions for QAM). Anyway, it just feels duplicated information, but I'm not objecting, just raising an opinion :)